src/remount-fs/remount-fs.c

```diff
@@ -69,11 +70,46 @@ static int do_remount(const char *path, bool force_rw, Hashmap **pids) {
         return track_pid(pids, path, pid);
 }
 
+static int mount_point_matches(const struct mntent *me) {
```

Comment:
instead of adding more infra based on glibc "struct mntent" apis, please switch to libmount